content/_index.html

```diff
@@ -5,89 +5,176 @@
-			<object type="image/svg+xml" data="multi-framework.svg" alt="Multi-framework">
-			</object>
-		</div>
-		<div class="text">
-			<h4>Multi-framework</h4>
-			<p>Our development plans go beyond TensorFlow, and we are working hard to include PyTorch, MXNet, Chainer, and more. We also integrate with Ambassador for ingress, Nuclio as a fast multi-purpose serverless framework, and Pachyderm for managing your data science pipelines.</p>
-		</div>
-	</div>
-
-	<div class="lead">
-		<div class="image right">
-			<img src="icn-people.svg" alt="People Icon">
-		</div>
-		<div class="text">
-			<h4>Community</h4>
-			<p>We are an open and welcoming community of software developers, data scientists, and organizations that are working to make it easier to develop and deploy scalable ML workflows across the industry. <a href="/docs/about/contributing/">Start contributing.</a>
-		</div>
-	</div>
+  <div class="container">
+    <div class="card-deck">
+      <div class="card border-primary">
+        <img
+          src="logos/Jupyter.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Jupyter Logo"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Notebooks</h5>
+          <p class="card-text text-light">
+            Kubeflow includes services to create and manage interactive <a target='_blank' class='text-light' href="https://jupyter.org/">Jupyter
+            notebooks</a>. You can choose custom images, and allocate specific
+            compute resources based on your data science needs. This
+            comprehensive experience allows you to create workflows locally, and
+            deploy them with ease.
+          </p>
+        </div>
+      </div>
+      <div class="card border-primary">
+        <img
+          src="logos/TensorFlow.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="TensorFlow"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">
+            TensorFlow model training
+          </h5>
+          <p class="card-text text-light">
+            Kubeflow has a built-in <a target='_blank' class='text-light' href="https://www.tensorflow.org/">TensorFlow </a> job operator that you can use to
+            train a machine learning model with TensorFlow. The training
+            controller can be configured to use various compute resources (CPUs
+            or GPUs) and is compatible with various cluster sizes.
+          </p>
+        </div>
+      </div>
+      <div class="card border-primary">
+        <img
+          src="logos/Models.png"
+		  class="card-img-top"
+		  draggable="false"
+          style="padding: 2rem;"
+          alt="Model Serving image"
+        />
+        <div class="card-body bg-primary">
+          <h5 class="card-title text-light section-head">Model Serving</h5>
```

Comment:
Change `Model Serving` to `Model serving` - use sentence case for consistency with the other headings on the page.